src/main/resources/META-INF/rewrite/spring-boot-30.yml

```diff
@@ -200,3 +201,30 @@ recipeList:
       methodPattern: org.springdoc.core.models.GroupedOpenApi.Builder addOpenApiCustomiser(..)
       newMethodName: addOpenApiCustomizer
       matchOverrides: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.spring.boot3.MigrateDropWizardDependencies
+displayName: Migrate dropWizard dependencies to Spring Boot 3.x
+description: Migrate dropWizard dependencies to the new artifactId, since these are changed with Spring Boot 3.
+tags:
+  - spring
+  - boot
+  - dropwizard
+recipeList:
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: com.codahale.metrics.servlet
+      newPackageName: io.dropwizard.metrics.servlet
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: com.codahale.metrics.servlets
+      newPackageName: io.dropwizard.metrics.servlets
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: io.dropwizard.metrics
+      oldArtifactId: metrics-servlet
+      newArtifactId: metrics-jakarta-servlet
+      newVersion: 4.2.xx
```

Comment:
I don't think I've seen `xx` in a version selector before. I think it would probably better as just `x` unless I am missing something here.

Reply:
@knutwannheden in the maven central, dropwizard version being specified in the format of 4.2.23 hence have given format as 4.2.xx. 

Please suggest.

I am happy to change, please create issue and assigned to me, I will work on it.

Thanks,
Mahi